footer.js

```diff
@@ -33,7 +33,8 @@ document.write('\
                 </div>\
             </div>\
             <div class="ui inverted section divider"></div>\
-            <img src="./assets/images/kosslogo.png" class="ui centered mini image">\
+            <img src="../assets/images/kosslogo.png" class="ui centered mini image" alt="">\
+            <img src="./assets/images/kosslogo.png" class="ui centered mini image" alt="">\
```

Comment:
@harishnandan Kindly remove this line. The KOSS logo is appearing twice in the front page.